projects/optic/src/commands/capture/sources/captured-interactions.ts

```diff
@@ -57,8 +57,11 @@ export class CapturedInteraction {
         requestPostData.text,
         requestPostData.encoding as BufferEncoding | undefined
       );
+
+      const asText = new TextDecoder().decode(buffer);
+
       requestBody = CapturedBody.from(
-        buffer,
+        asText,
```

Comment:
This is the big change -- instead of creating a buffer JUST to stream it later back into text, we use the buffer's built in encoding to parse the string data in the HAR base64 or gzip into a normal string.